sections/main-product.liquid

```diff
@@ -278,6 +279,11 @@
                         {%- else -%}
                           {{ 'products.product.sold_out' | t }}
                         {%- endif -%}
+                        <div class="loading-overlay__spinner hidden">
+                          <svg aria-hidden="true" focusable="false" role="presentation" class="spinner" viewBox="0 0 66 66" xmlns="http://www.w3.org/2000/svg">
+                            <circle class="path" fill="none" stroke-width="6" cx="33" cy="33" r="30"></circle>
+                          </svg>
+                        </div>
                     </button>
                     {%- if block.settings.show_dynamic_checkout -%}
                       {{ form | payment_button }}
```

Comment:
The reason we will want to have the collage with `grid` is because we want to be able to reorder the blocks and we want the 1st or last item to take up 2/3 of the space and the last two to take 1/3.

We could use `flex-basis:33.33%;` and 66.66%, but we wouldnt be able to stack the last two or first two items